Makefile

```diff
@@ -56,42 +56,55 @@ endif
 all: install
 
 checks:
-	@echo "Checking deps:"
+	@echo -n "Check deps: "
 	@(env bash $(PWD)/buildscripts/checkdeps.sh)
+	@echo "Done."
+	@echo -n "Checking project is in GOPATH: "
 	@(env bash $(PWD)/buildscripts/checkgopath.sh)
+	@echo "Done."
 
 getdeps: checks
-	@echo "Installing golint:" && go get -u github.com/golang/lint/golint
-	@echo "Installing gocyclo:" && go get -u github.com/fzipp/gocyclo
-	@echo "Installing deadcode:" && go get -u github.com/remyoudompheng/go-misc/deadcode
-	@echo "Installing misspell:" && go get -u github.com/client9/misspell/cmd/misspell
-	@echo "Installing ineffassign:" && go get -u github.com/gordonklaus/ineffassign
+	@echo -n "Installing golint: " && go get -u github.com/golang/lint/golint
```

Comment:
I have faced this problem. I don't know if it bothers many. I can do without using `make` when offline.